src/goto-instrument/cover.cpp

```diff
@@ -741,7 +1002,7 @@ std::map<exprt, signed> values_of_atomic_exprs(
   for(auto &c : conditions)
   {
     std::set<signed> signs=sign_of_expr(c, e);
-    if(signs.size()==0)
+    if(signs.empty())
```

Comment:
I like those changes, but it would be great if they were in a separate commit or even pull request as the commit message of this one is not going to explain very well why they had been made.